cicd/pipelines/using-pipelines-as-code.adoc

```diff
@@ -6,18 +6,18 @@ include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-// :FeatureName: Pipelines as Code
 
+// :FeatureName: Pipelines as Code
 [role="_abstract"]
-With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, the feature runs the pipeline and reports the status.   
+With {pac}, cluster administrators and users with the required privileges can define pipeline templates as part of source code Git repositories. When triggered by a source code push or a pull request for the configured Git repository, {pac} runs the pipeline and reports the status.
 
 [id="pac-key-features"]
 == Key features
 {pac} supports the following features:
 
 * Pull request status and control on the platform hosting the Git repository.
 * GitHub Checks API to set the status of a pipeline run, including rechecks.
-* GitHub pull request and commit events. 
+* GitHub pull request and commit events.
```

Comment:
For lists that only use sentence fragments, do not use periods at the end, per IBM style guide: https://www.ibm.com/docs/en/ibm-style?topic=punctuation-periods#lists

slightly outside the scope of this PR, so if you want to open a Jira to follow this up later that's maybe OK, but it would be nice if the periods could be removed from the list in this PR

Reply:
Style reviews throughout this document  will be a part of the restructure that is planned for the next version. no Kira required - time is of the essence and I'd like to avoid increasing PR scope.